code/modules/mob/living/silicon/robot/drone/drone_items.dm

```diff
@@ -374,4 +383,4 @@
 					stack = stack_plastic
 
 			stack.amount++
-			decompiler.stored_comms[type]--;
\ No newline at end of file
+			decompiler.stored_comms[type]--;
```

Comment:
Oh sure. It was there by default. I don't know why it shows that I changed the line.